test/test_images/probe_helper/probe_helper.go

```diff
@@ -249,17 +258,9 @@ func (ph *ProbeHelper) forwardFromProbe(ctx context.Context) cloudEventsFunc {
 				}
 			}
 		case CloudAuditLogsSourceProbeEventType:
-			switch event.Subject() {
-			case CloudAuditLogsSourceProbeCreateSubject:
-				// Create a pubsub topic with the given ID.
-				if _, err := ph.pubsubClient.CreateTopic(ctx, event.ID()); err != nil {
-					return logNACK(ctx, "Probe forwarding failed, error creating pubsub topic %v: %v", event.ID(), err)
-				}
-			case CloudAuditLogsSourceProbeDeleteSubject:
-				// Delete the pubsub topic with the given ID.
-				if err := ph.pubsubClient.Topic(event.ID()).Delete(ctx); err != nil {
-					return logNACK(ctx, "Probe forwarding failed, error deleting pubsub topic %v: %v", event.ID(), err)
-				}
+			// Create a pubsub topic with the given ID.
+			if _, err := ph.pubsubClient.CreateTopic(ctx, event.ID()); err != nil {
```

Comment:
I think the topic name should be "cloudauditlogssource-probe-"+event.ID()?

Reply:
The event ID already includes the probe type, so event.ID() looks something like "cloudauditlogssource-probe-1234567890"